examples/publish_wave.py

```diff
@@ -45,11 +38,9 @@ def on_participant_disconnect(participant: livekit.Participant, *_):
     logging.info("connecting to %s", URL)
     try:
         e2ee_options = livekit.E2EEOptions()
-        e2ee_options.key_provider_options.shared_key = b"password"
 
         await room.connect(URL, TOKEN, options=livekit.RoomOptions(
             auto_subscribe=True,
-            e2ee=e2ee_options
```

Comment:
can we leave these two lines there, but just to comment out `e2ee` here.

Reply:
I think we already have the e2ee.py example for that, I don't think this is necessary